lib/Amazon/MiqEc2InstanceTypes.rb

```diff
@@ -36,6 +38,7 @@ module MiqEc2InstanceTypes
       :ebs_optimized_available => nil,
       :enhanced_networking     => nil,
       :cluster_networking      => nil,
+      :cloud_subnet_required   => true,
```

Comment:
I know I originally told you to use cloud_subnet here, but in hindsight, I think this should say vpc_available in the raw data.  The raw data here is Amazon specific.  We should do the "translation" in the parser.